src/Filter/types/index.ts

```diff
@@ -415,6 +416,14 @@ export interface MultiSelectFilterProps {
  */
  export interface MultiSelectFilterOptions extends Omit<Partial<FilterModule<string[]>>, 'description' | 'label'> {}
 
+/**
+ * `ParseMultiValue` is the structure of the option to allow parsing multiple values from a delimited list.
+ */
+export interface ParseMultiValue {
+    enabled: boolean;
+    delimiter: string;
```

Comment:
can the delimiter be optional, and default to `\n`?

Reply:
@amirkibbar  - As mentioned above, getting rid of `enabled` would allow me to hike the `delimiter` up a level and make parsing dependent on if it is specified. e.g.
```js
export interface MultiSelectFilterProps {
    options: MultiSelectOption[];
    label: string;
    description?: string;
    initialValue?: any[];
    creatable?: boolean;
    handleCreateItem?: (item: string) => void;
    disableMenu?: boolean;
    delimiter?: Delimiter;
}
```
Doing so would allow for two different approaches.

1. `type Delimiter = string;` - This approach would be the simplest to implement and requires the user to specify a delimiter. I prefer this one.
2. `type Delimiter  = true | string;` - If the user specifies `true`, we could default to using `\n`. And of course if they insert a string we'll use that instead. While this would work, I feel like the user  might as well just specify the delimiter they want since they have to specify `true` anyway. And this also puts a large assumption that anyone consuming the application would want `\n` as their most used delimiter.

Thoughts?

Reply:
let's not over complicate it - `string` should do (option 1). If the user provides a delimiter then the feature should be turned on and the delimiter used. Otherwise, the feature if off. No need for a default delimiter

Reply:
Sounds good. Thanks!

Reply:
done